libsrc4/nc4var.c

```diff
@@ -1708,3 +1708,87 @@ NC4_get_vara(int ncid, int varid, const size_t *startp,
    return nc4_get_vara(nc, ncid, varid, startp, countp, memtype,
                        0, (void *)ip);
 }
+
+
+/* Provide a temporary hook
+to choose old NCDEFAULT methods vs new versions
+of get/put vars
+*/
+/* Temporary flag to choose default vs not put/get vars */
+static int defaultvars = 0;
+
+void
+nc4_set_default_vars(int tf)
+{
+    defaultvars = (tf ? 1 : 0);
+}
```

Comment:
Can this temporary flag be removed?

Changing to the old default methods can easily be done by changing the dispatch table.